runtime/laos/src/currency.rs

```diff
@@ -0,0 +1,31 @@
+use laos_primitives::Balance;
+
+// Unit = the base number of indivisible units for balances
+// 18 decimals
+pub const UNIT: Balance = 1_000_000_000_000_000_000;
+pub const MILLIUNIT: Balance = UNIT / 1000;
+pub const MICROUNIT: Balance = MILLIUNIT / 1000;
+
+// Define storage fees as constants for clarity and reuse
+const STORAGE_ITEM_FEE: Balance = 10 * UNIT;
+const STORAGE_BYTE_FEE: Balance = 10 * MILLIUNIT;
+
+/// Calculates the deposit required based on the number of items and bytes.
+pub(crate) const fn calculate_deposit(items: u32, bytes: u32) -> Balance {
```

Comment:
A question, maybe just for me to understand it better. But calculate_deposit is not part of the proxy pallet configuration? Should it be on the proxy config file??

Reply:
it's used also in multisig configuration. I moved it under currency mod following the pattern used in Moonbeam: https://github.com/moonbeam-foundation/moonbeam/blob/b2b9dfeff44dc07ce407b224a98159e33a3e073f/runtime/moonbeam/src/lib.rs#L143